Src/SmtpServer/ComponentModel/ServiceProvider.cs

```diff
@@ -11,7 +11,7 @@ namespace SmtpServer.ComponentModel
     /// </summary>
     public sealed class ServiceProvider : IServiceProvider
     {
-        public static readonly IServiceProvider Default = new ServiceProvider();
+        internal static readonly IServiceProvider Default = new ServiceProvider();
```

Comment:
These should stay public, not internal.. the idea is that these can be the default impl for anyone to use